flutter_test_app/ios/intentcall_codegen.sh

```
@@ -1,4 +1,6 @@
 # intentcall-platform: begin
 cd "${SRCROOT}/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)